ceph.spec.in

```diff
@@ -292,6 +292,7 @@ BuildRequires:  libevent-devel
 %endif
 %if 0%{with system_pmdk}
 BuildRequires:  ndctl-devel
+BuildRequires:  daxctl-devel
```

Comment:
Could we change the commit message? //I cannot post in the commit message part:

In order to support the character device of pmem usage in bluestore via libpmem built by Ceph itself,  we need to enable daxctl and ndctl dependency.  And this pr is used to add  t eh daxctl dependency,

Reply:
done

Reply:
just a nit, if we have requirement regarding to the version of a library, might want to put something like
```
BuildRequires: ndctl-devel >= 63
BuildRequires: daxctl-devel >= 63
```

Reply:
... and same for `libpmemobj-devel` (based on `libpmemobj-dev (>= 1.8)` in `debian/control`)?